iOS/README.md

```diff
@@ -0,0 +1,42 @@
+# PlayFab Multiplayer SDK for iOS
+
+## Logging
+
+The PlayFab Multiplayer C++ library includes logging capabilities with a configurable verbosity level. Logging configuration is defined in `PlayFabMultiplayerLogger.json` file which can be deployed as an asset along with the application.
+
+When this file is detected by the application at runtime it will use it to enable logging as configured. The following verbosity levels are currently supported:
+1. `VERBOSE` - everything
+2. `INFO` - less than everything, only important messages and errors
+3. `ERROR` - only errors
+
+The logging is disabled by default, but can be enabled with "enabled" property set to `true`.
+
+### Example configuration file
+
+```json
+{
+    "enabled": false,
+    "bufferSize": 262144,
+    "maxNumberOfItemsInBatch": 500,
+    "maxBatchWaitTimeInSeconds": 1,
+    "readBufferWaitTimeInMilliseconds": 1,
+    "logFolder": "/app_sandbox_storage/Documents/",
+    "logLevel": "VERBOSE",
+    "muLogEnabled": false,
+    "maxLogFileSizeInMegabytes": 0,
+    "logAreaFlags_ApiInOut": -1,
+    "logAreaFlags_FnInOut": 18056,
+    "logAreaFlags_Log": 20104
+}
+```

```

Comment:
is there any documentation anywhere about what these different fields are and what values they can take?

for example, it's not clear to me what `-1` means for ApiInOut vs `18056` for FnInOut

Reply:
Do you mean for developers? no, 
These are the masks which are manually calculated based on internal enum defines, sometimes we provide them to titles if we need specific component logs but not other.

Reply:
ok. I think this relates to my other comment about documenting that these values are numeric and trying to make these examples use reasonable defaults for the PFMultiplayer library

then we can just add comments to the example saying these values are internal just so no one expects that they should know what the values mean or how to modify them, themselves

Reply:
We've never had any studio ask us about this value, but its good to add the line there that its internal.